Barotrauma/BarotraumaShared/SharedSource/Items/Components/Machines/Controller.cs

```diff
@@ -277,7 +277,7 @@ public override bool Use(float deltaTime, Character activator = null)
                 return false;
             }
 
-            item.SendSignal(0, "1", "trigger_out", user);
+            item.SendSignal("1", "trigger_out");
```

Comment:
The user should be passed to item.SendSignal here

Reply:
I thought I'd caught all of these - evidently I missed one. Fixed in 3f50c6a8bfbd095d4f231e2fbd11986ff072fc37.